app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,69 @@
+/* mixins, variables, etc. */
```

Comment:
Not sure thins file is the best place to put this stuff. But know we dont have a better place right, now.  Lets keep this in mind, for best CSS architecture.